src/oracles/oracles.ts

```diff
@@ -18,6 +18,11 @@ export enum OracleType {
   WSTETH_ORACLE,
   BOUNDED_ORACLE,
   COMPOSITE_ORACLE,
+  AAVE_ORACLE,
```

Comment:
Please, double check that numbers are the same, or  you can assign these numbers directly